public/stylesheets/README.md

```diff
@@ -21,3 +21,5 @@ process as all other code changes.
 Remember, though: changes to a skin's files in the masters directory will not
 affect the skin on production! An admin will still need to edit the skin via
 the admin interface and paste in the updated master version.
+
+**Note**: To keep development data in sync with the master copies, please ensure any skin changes are mirrored in `site/user_skins_to_load`.
```

Comment:
This isn't a good idea -- it's simply too easy for things to get out of sync. We need an approach where we will only have one copy of each skin and then load them into the database from there. It's okay if that means adding code comments to the master copies, renaming the files, and/or moving them to a different directory, as long as we also keep the subdirectory structure. 

(I don't think we need to stick with the `user_skins_to_load` name, either. It's an unintuitive name that makes it sound like we're loading skins for individual users rather than loading the officially supported skins for the chooser.)